__tests__/app.test.js

```diff
@@ -123,3 +123,40 @@ describe("task 4. /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("/api/articles/:article_id/comments", () => {
+  test("GET 200, returns array of comments of passed as article_id article", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body: { comments } }) => {
+        expect(comments.length).toBe(11);
+        expect(comments[0].comment_id).toBe(5);
```

Comment:
The comment id will not be pertinent to the article, ideally this check should be for the article id that the comment is linked to. A nice forEach here to check that key on each object would be solid.